openedx/core/djangoapps/schedules/management/commands/tests/send_email_base.py

```diff
@@ -169,13 +175,27 @@ def test_schedule_bin(self, schedule_count, mock_metric, mock_ace):
             self.assertEqual(mock_schedule_send.apply_async.call_count, schedule_count)
             self.assertFalse(mock_ace.send.called)
 
+    def _schedule_factory(self, offset=None, **factory_kwargs):
```

Comment:
**Nit:** Can we move `_schedule_factory` towards the top of the class along with the other helper methods - and before all the test methods are listed?

Reply:
Done!